pkg/storage/postgresql/stats.go

```diff
@@ -488,6 +503,9 @@ func (db DB) StatsCurrentExposure(filter storage.Filter) (model.StatsExposure, e
         {{ if .Tag }} 
         AND tt.tag = :tag
         {{ end }}
+		{{ if .Tags }}
```

Comment:
I noticed this SQL statement was indented with spaces and looks wrong with the new line indented with tabs.